api_server.py

```
@@ -80,32 +80,56 @@ def get_crew_list() -> List[Dict]:
         return [dict(r) for r in rows]
 
 
+def format_availability_data(end_time_val: Any, now: datetime) -> Dict:
```

Ruff (0.15.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `end_time_val`

(ANN401)